docs/docs/quick-start.md

```diff
@@ -1,66 +1,66 @@
 ---
-title: Quick Start
+title: Início Rápido
 ---
 
-This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, [head to our tutorial](/tutorial/)!
+Esse guia de início rápido é direcionado para desenvolvedores com conhecimento intermediário ou avançado. Para uma introdução mais amigável ao Gatsby, [visite nosso tutorial](/tutorial/)!
 
-## Use the Gatsby CLI
+## Usando o Gatsby CLI
 
 <EggheadEmbed
   lessonLink="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-create-develop-and-build-gatsby-sites-from-the-command-line"
   lessonTitle="Quick Start with Gatsby: Create, Develop, and Build Gatsby Sites From the Command Line"
 />
 
-**Note**: this video uses `npx`, which is a tool to execute an npm package without first installing it. Running the command `npx gatsby new` is the same as running `gatsby new` after installing the gatsby-cli on your computer.
+**Nota**: este vídeo usa o `npx`, que é uma ferramenta para executar um pacote npm sem primeiro instalá-lo. Executar o comando `npx gatsby new` é o mesmo que executar `gatsby new` após instalar o gatsby-cli no seu computador.
 
-### Install the Gatsby CLI.
+### Instalando o Gatsby CLI.
 
 ```shell
 npm install -g gatsby-cli
 ```
 
-### Create a new site.
+### Criando um novo site.
 
 ```shell
 gatsby new gatsby-site
 ```
 
-### Change directories into site folder.
+### Mudando o diretório para a pasta do site.
 
 ```shell
 cd gatsby-site
 ```
 
-### Start development server.
+### Iniciando o servidor de desenvolvimento.
 
 ```shell
 gatsby develop
 ```
 
-Gatsby will start a hot-reloading development environment accessible by default at `localhost:8000`.
+O Gatsby iniciará um ambiente de desenvolvimento com *hot-reloading*, que pode ser acessado por padrão em `localhost:8000`.
```

Comment:
Para manter coerência com os outros arquivos:

```suggestion
O Gatsby iniciará um ambiente de desenvolvimento com _hot-reloading_, que pode ser acessado por padrão em `localhost:8000`.
```

Reply:
Algum motivo pelo qual trocar o caractere? Os dois indicam expressões em itálico.